src/main/java/de/tudarmstadt/ukp/wikipedia/parser/mediawiki/ModularParser.java

```diff
@@ -1074,7 +1074,8 @@ private void convertGalleriesToImages(SpanManager sm, List<Span> tagSpans)
                     for (String imageParagraph : tokenize(sm, startSpan.getEnd(), endSpan
                         .getStart(), lineSeparator))
                     {
-                        String[] splitImageParagraph = imageParagraph.split("\\|");
+						String[] splitImageParagraph = imageParagraph.split("\\|", -1);
+
```

Comment:
That's because original code had tabs, but indenting with tabs is a bad practice.

I also didn't want to make the diff unreadable, so in a separate PR I'll do some reformatting.